src/wolfgame.py

```diff
@@ -5799,12 +5803,28 @@ def see(cli, nick, chan, rest):
             victimrole = var.DEFAULT_ROLE
             if var.DEFAULT_SEEN_AS_VILL:
                 victimrole = "villager"
+        if victim in var.DECEIVED:
+            if victimrole == "wolf":
+                victimrole = "villager"
+            else:
+                victimrole = "wolf"
+
+        if nick in var.DECEIVED: # it DOES stack! so if both victim and seer have the totem, it's canceled out
+            if victimrole == "wolf":
+                victimrole = "villager"
+            else:
+                victimrole = "wolf"
+
         pm(cli, nick, (messages["seer_success"]).format(victim, victimrole))
         debuglog("{0} ({1}) SEE: {2} ({3}) as {4}".format(nick, role, victim, vrole, victimrole))
     elif role == "oracle":
         iswolf = False
         if (victimrole in var.SEEN_WOLF and victimrole not in var.SEEN_DEFAULT) or victim in var.ROLES["cursed villager"]:
             iswolf = True
+        if victim in var.DECEIVED:
+            iswolf = not iswolf
+        if nick in var.DECEIVED: # do *not* merge these two ifs - they stack
+            iswolf = not iswolf
```

Comment:
Sure. `victim in var.DECEIVED` would be a bool already do, so just `victim in var.DECEIVED ^ nick in var.DECEIVED` (possibly with a comment explaining why).